### ISSUE TITLE
[object_detection] Does the OD API supports single-stage segmentation?

### ISSUE THREAD

**Issue by netanel-s** [tensorflow/models#8055]
### System information
- **What is the top-level directory of the model you are using**: object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubunto 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.13
- **Bazel version (if compiling from source)**: -
- **CUDA/cuDNN version**: CUDA 9
- **GPU model and memory**: Titan RTX 23 GB
- **Exact command to reproduce**: not relevant

### Describe the problem
I see one can configure a masks predictions head in an SSD model as can be seen [here](https://github.com/tensorflow/models/blame/master/research/object_detection/protos/ssd.proto#L103). But I don't find in the code how it is being used, and whether and how it contributes to the loss function.
I know the API supports Mask RCNN, but I need to use the single-stage variant instead.
Please share any insights regarding how the API produces the mask predictions and where is the code which wires it to the total loss.
If it is not - is is expected?

Thanks in advance.
